gui/packages/ubuntupro/lib/core/settings.dart

```diff
@@ -12,9 +12,9 @@ class Settings {
     final purchase = repository.readInt(kAllowStorePurchase) == 1
         ? Options.withStorePurchase
         : Options.none;
-    final landscape = repository.readInt(kShowLandscapeConfig) == 1
```

Comment:
You could have kept Show and check if it's set to 0. If we decide to change the default again (which is totally possible), it woud avoid to change the name of the key again. What do you think?

Reply:
Besides I have the powers of Git, so if there comes a time we need to go back I can simply `git revert ...`.

Reply:
Found a longer, but better, name for the value. I just rebased over latest main. There is only one new commit.